mlflow/models/model.py

```diff
@@ -75,6 +76,11 @@
 SET_MODEL_ERROR = (
     "Model should either be an instance of PyFuncModel, Langchain type, or LlamaIndex index."
 )
+ENV_VAR_FILE_NAME = "environment_variables.txt"
+ENV_VAR_FILE_HEADER = (
+    "# This file records environment variable names that are used during model inference\n"
+    "# make sure you set them when creating a serving endpoint from this model.\n"
```

Comment:
`make sure you set them` sounds like the environment variables in this file are required but we're not sure. I think we should mention that here to avoid confusion.